<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionCloseSystem"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="actionAlter"/>
   <addaction name="actionSearch"/>
   <addaction name="actionPrint"/>
   <addaction name="actionPrintPreview"/>
   <addaction name="actionPdf"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-open.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-save.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-print.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-print.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭文件</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/list-remove.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/add.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/add.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actionAlter">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/edit-find.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/stock_zoom-page.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/stock_zoom-page.png</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionCloseSystem">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/application-exit.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>关闭系统</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-print-preview.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-print-preview.png</iconset>
   </property>
   <property name="text">
    <string>打印预览</string>
   </property>
  </action>
  <action name="actionPdf">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/stock_new-tab.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/stock_new-tab.png</iconset>
   </property>
   <property name="text">
    <string>另存为pdf</string>
   </property>
  </action>
  <action name="actionSacveAs">
   <property name="text">
    <string>saveAs</string>
   </property>
  </action>
  <action name="actionPicture">
   <property name="text">
    <string>另存为图片</string>
   </property>
  </action>
  <action name="actionExcel">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-save-as.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>另存为excel</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/gtk-help.png</normaloff>:/new/prefix1/C:/Users/Herbert/Desktop/课程资料/C++软件实习/图标/gtk-help.png</iconset>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
